### ISSUE TITLE
The process ends after the MAA is connected to emulator.

### ISSUE THREAD

**Issue by Eundong** [MaaAssistantArknights/MaaAssistantArknights#5269]
### Before raising the issue...

- [X] I understand that Issues are for feedback and problem solving, not trolling the comments section, and will provide as much information as possible to help solve the problem.
- [X] I filled in a short, clear title so that developers could quickly identify the general problem when going through the issue list. Instead of "Some suggestions", "Stuck", etc.
- [X] I have checked the latest beta update and there is no mention of the bug being fixed.

### Description

After running the MAA and connecting to the emulator, the process ends after a certain time (2~3 seconds). 
This happens similarly even when executing various requests.
When the process has ended, there were same log message in asst.log file

[2023-06-28 06:09:19.203][TRC][Pxa64][Tx100c] asst::OcrPack::check_and_load | enter

After this log is recorded, the process is terminated. I've tried changing the touch mode, but nothing changed. it seems that the problem fundamentally occurs in the OcrPack.

### Log files

[asst.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/11887378/asst.log)
[gui.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/11887379/gui.log)

**Comment by Eundong** [comment]
I think that this issue occurred after OCR-related updates. It seems there was a recent OCR update, and when I test with version 4.19.0, it operates normally. crash not happened

**Comment by ABA2396** [comment]
I am unable to reproduce this issue on my end. Please try downloading the complete package again to see if it resolves the problem. It's possible that the updated compressed file may be corrupted.

**Comment by Eundong** [comment]
Thank you for the quick reply.
I downloaded the complete package and tried again, only to confirm the same error repeating.
I strongly suspect the issue may be with the OCR program, so I tested the OCR file under the /resource/ folder.

As a result, I found that when I replaced the /resources/PaddleOCR/det/inference.onnx from version 4.19.0 in the same path of the latest version, the issue did not occur.
I found there are patch about that file two weeks ago. https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/260e939b057d08a8883c1f1de0521fc2a7ebf361
It appears there is a problem with this particular patch.

It seems I forgot to mention this in my first report, I am using a Yostar-KR server.
